paper.md

```diff
@@ -207,10 +207,29 @@ It can either be created by users or provided by system administrators.
 
 # Case Study {#sec:study}
 
-### **Single-Node local workflow execution**
+In this section, we present three case studies of how the Popper workflow engine allows reproducing and scaling workflows easily in different compute environments.
+<!-- briefly describe each case studies here -->
+
+### **Workflow execution on the local machine**
+
+In this section, we discuss how Popper helps researchers to write and test workflows on their local machines and iterate fast.
+We used an Apple Macbook Pro with a 64-bit Intel i5 processor and 8Gb RAM for this casestudy.
+An Image classification workflow was built and run on the MNIST dataset [@deng2012mnist] using the Docker container engine.
+On single node machines, Popper leaves the job of scheduling the containerized steps to the host machines OS.
+We ran the workflow with an overfitting patience of 5 on the laptop's CPU.
+It took around 4035 seconds for the workflow to reach an accuracy of 99.19% over 12 epochs, averaged over 5 test runs.
+To achieve lower training durations, the training should ideally be done on GPU's in the cloud which inturn requires these workflows to be easily portable to multi-node cloud environments.
```

Comment:
we're missing more detail on multiple aspects:

  * how the workflow looks like, how many steps does it have, what does each step do? We could add the actual YAML file. We might want to have a subsection where we describe this workflow and what it does.

  * a quick background section on the tools being used.

  * point to the fact that we are using multiple docker images, and that while we could put all the dependencies in the same image, doing so would increase the complexity of handling dependencies and defeat the goal of using containers.

  * I think we also are missing a description of the general workflow of popper usage from an experimenter point of view. This might be more suitable for the previous section, but in general the workflow would be:

      * think of the steps of the workflow
      * for each step, determine which software is needed and then:
           + search for an available image that already has this software installed. For this, we explain how this is done and where do users look for this.
           + If no image is found, a `Dockerfile` needs to be created. So this would give a quick overview of what `Dockerfile`s are.
      * run the workflow and refine it.

  * Same as above but for how Popper runs a workflow. Kind of a "life of a workflow execution". This explaining the tasks that a runner carries out in order to execute a step: build or pull container; create and configure container appropriately; run container

Reply:
@ivotron By `a quick background section on the tools being used`, do you mean explaining keras, mnist, GPU etc, ?

Reply:
yes, in general a description of what the workload does, with an emphasis of how difficult it is to reproduce these locally. For example, we can point to mlperf benchmarks and maybe do a quick analysis of the issues that get opened where people are asking about how to reproduce a benchmark execution (there are many of those in the mailing list as well)

Reply:
@ivotron  I am thinking about this last message. Maybe you can take a look at the rest of the changes. Thanks a lot.